perses/rjmc/geometry.py

```diff
@@ -632,7 +624,7 @@ def _normalize_torsion_proposal(self, atom, r, theta, bond_atom, angle_atom, tor
             ub[i]+=ub_i
 
         #exponentiate to get the unnormalized probability
-        q = np.exp(-ub)
+        q = np.exp(ub)
```

Comment:
I'm confused by `exp(ub)`.  Is `ub` supposed to be an energy?  If not, maybe you could just call this `logq` instead of `ub`?